src/js/state.js

```diff
@@ -41,6 +41,14 @@ export default async function () {
   } else {
     ciphers = ciphers;
   }
+  if (configs[server].usesOpenssl !== false && minver('3.0.0', form['openssl'].value) === true) {
+    // set SECLEVEL=0 for openssl 3 to support TLSv1 and TLSv1.1 for Old config
+    if (protocols.includes('TLSv1.1')) {
+      if (ciphers[0] !== '@SECLEVEL=0') {
+        ciphers.unshift('@SECLEVEL=0');
+      }
+    }
+  }
```

Comment:
is the condition broken down for linting (line/length) reasons? or a simple:

```suggestion
  if (configs[server].usesOpenssl && protocols.includes('TLSv1.1') && minver('3.0.0', form['openssl'].value)) {
    // set SECLEVEL:0 via cipher string to support TLSv1-1.1 "old" with OpenSSL 3.x
    if (!ciphers.includes('@SECLEVEL=0')) ciphers.unshift('@SECLEVEL=0');
  }
```

could do?

Reply:
Well, empty ciphers may happen if there's no overlap of the recommendation with the ciphers available from the server.  So maybe even:
```suggestion
  if (configs[server].usesOpenssl && protocols.includes('TLSv1.1') && minver('3.0.0', form['openssl'].value) && ciphers.length) {
    // set SECLEVEL:0 via cipherstring to support TLSv1-1.1 "old" with OpenSSL 3.x
    if (!ciphers.includes('@SECLEVEL=0')) ciphers.unshift('@SECLEVEL=0');
  }
```
?

Reply:
> is the condition broken down for linting (line/length) reasons?

Yes.

> Well, empty ciphers may happen if there's no overlap of the recommendation with the ciphers available from the server.

Not with the current "Old" config.  I expect any update to the "Old" config which still includes TLSv1 or TLSv1.1 will include a non-empty ciphers list.

In any case, I incorporated some of your changes, but still avoided excessively long lines.

Reply:
This is not related to `old` config, but the intersection with servers' support in `configs[server].supportedCiphers` that may end up empty, there's no guardrail around none overlapping (as it's then checked in the outputs)